plumbing/object/commit.go

```diff
@@ -46,6 +46,8 @@ type Commit struct {
 	TreeHash plumbing.Hash
 	// ParentHashes are the hashes of the parent commits of the commit.
 	ParentHashes []plumbing.Hash
+	// Encoding is the encoding of the commit.
+	Encoding string
```

Comment:
Please create a type that represents this, and also its default value (a.k.a `UTF-8`).

```suggestion
	Encoding MessageEncoding
```